runtime/beta/errs/error.go

```diff
@@ -141,6 +141,8 @@ func (e *Error) Unwrap() error {
 
 func HTTPError(w http.ResponseWriter, err error) {
 	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set("X-Content-Type-Options", "nosniff")
```

Comment:
What's this? Probably deserves a comment

Reply:
From: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Content-Type-Options
> The X-Content-Type-Options response HTTP header is a marker used by the server to indicate that the [MIME types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types) advertised in the [Content-Type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) headers should be followed and not be changed. The header allows you to avoid [MIME type sniffing](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types#mime_sniffing) by saying that the MIME types are deliberately configured.

Basically, I noticed that `http.Error` had this in it, but our `errs.HTTPError` did not. Seemed like a good idea to bring it over from the standard library.

Reply:
Ah nice!